### ISSUE TITLE
Parse method does not work on new iOS 16.4.1

### ISSUE THREAD

**Issue by mdeimert** [bottlepay/flutter_libphonenumber#42]
Parse method does not work on 16.4.1 iOS getting this error:

> Domain=NSCocoaErrorDomain Code=4099 "The connection to service with pid 1119 named com.apple.commcenter.coretelephony.xpc was invalidated from this process." UserInfo={NSDebugDescription=The connection to service with pid 1119 named com.apple.commcenter.coretelephony.xpc was invalidated from this process.}

I've tested the same build on iOS 16.4.0 and the plugin works fine.

**Comment by acoutts-nydig** [comment]
Whoa nice find! Looking at the phonenumberkit repo, it seems to be a known issue.

https://github.com/marmelroy/PhoneNumberKit/issues/626

I'll bump the pod version and get that pushed now.

**Comment by mdeimert** [comment]
Thanks a lot!

**Comment by acoutts-nydig** [comment]
I seem to be having an issue with cocoapods right now, across several projects so it must be a cocoapods CDN issue.

```[!] CDN: trunk URL couldn't be downloaded: https://cdn.cocoapods.org/CocoaPods-version.yml Response: URL using bad/illegal format or missing URL```

I might just need to wait a few hours, I'll give it another try later tonight / tomorrow morning.

**Comment by mdeimert** [comment]
This might help @acoutts-nydig :

> I got it working by using below commands:


`pod repo remove trunk`

> and then


`pod install or pod update`

**Comment by mdeimert** [comment]
Hi @acoutts-nydig  were you able to check it out? - it's important for my production app :( Sorry to bother.

**Comment by acoutts-nydig** [comment]
I just released `2.1.0` with the latest PhoneNumberKit - give that a try please.

Also note the breaking change that we need to manually include the dependency in our Podfile now. I filed this issue asking how we can work around it: https://github.com/marmelroy/PhoneNumberKit/issues/638

They stopped publishing updates to cocoapods and only put it on git now, and you can't define a git dependency in a podspec file.

**Comment by mdeimert** [comment]
Hi Thanks for the update.

Im getting this error now:

```
Could not build the precompiled application for the device.
Swift Compiler Error (Xcode): No such module 'PhoneNumberKit'
/Users/mike/.pub-cache/hosted/pub.dev/flutter_libphonenumber_ios-1.1.0/ios/Classes/SwiftFlutterLibphonenumberIosPlugin.swift:2:7
```

After updaing package i did flutter clean removed Podfile.lock and updated Podfile with the PhoneNumberKit line.

**Comment by mdeimert** [comment]
In think this podspec dependency still needs to be there:

![image](https://user-images.githubusercontent.com/41264745/235456386-815d80cb-a49b-448f-9ae5-f761089ff6d6.png)

I've added it manually on my local to `flutter_libphonenumber_ios.podspec` and the build goes smoothly and plugin works fine on 16.4.1. But we will need it on your master branch 👍

**Comment by acoutts-nydig** [comment]
aha! of course - have to include it in the podspec as well.

**Comment by mdeimert** [comment]
@acoutts-nydig  You will bump the version or it will be the same? :) Thanks in advance.

**Comment by acoutts-nydig** [comment]
Published 2.1.1 with the fix

**Comment by mdeimert** [comment]
Tested & works good on 16.4.1. Thank you for quick reaction!
